tfhe/src/integer/server_key/radix_parallel/tests_unsigned.rs

```diff
@@ -2962,3 +2963,41 @@ where
         );
     }
 }
+
+fn integer_trim_msb_blocks_handles_dirty_inputs<P>(param: P)
```

Comment:
you confirmed this test failed before the fix ?

Reply:
yep

Reply:
perfect thanks !